app/interactions/matches/create_match.rb

```diff
@@ -0,0 +1,178 @@
+    player = Player.create!(match: match, house: "Tyrell")
+    Boat.create!(player: player, board: match.board, territory: "redwyne_straights", x: 77, y: 2504)
+    Knight.create!(player: player, board: match.board, territory: "highgarden", x: 389, y: 2171)
+    Footman.create!(player: player, board: match.board, territory: "highgarden", x: 406, y: 2262)
+    Footman.create!(player: player, board: match.board, territory: "dornish_marches", x: 498, y: 2308)
+    player
+  end
+
+  def create_martell_player(match)
+    player = Player.create!(match: match, house: "Martell")
+    Boat.create!(player: player, board: match.board, territory: "sea_of_dorne", x: 1121, y: 2475)
+    Knight.create!(player: player, board: match.board, territory: "sunspear", x: 1010, y: 2600)
+    Footman.create!(player: player, board: match.board, territory: "sunspear", x: 1107, y: 2611)
+    Footman.create!(player: player, board: match.board, territory: "salt_shore", x: 814, y: 2698)
+    player
+  end
+
+  def create_tracks(match, board, players)
+    [
+      WildlingsTrack,
+      VictoryTrack,
+      # SupplyTrack,               # TODO
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
+  end
+
+  def distribute_player_things(match)
+    match.players.each do |player|
+      create_orders(player, match.board)
+      create_power_tokens(player)
+      send("create_#{player.house.downcase}_cards", player)
+    end
+  end
+
+  def create_orders(player, board)
+    March.create!(player: player, board: board)
+    MarchM.create!(player: player, board: board)
+    MarchP.create!(player: player, board: board)
+    ConsolidateP.create!(player: player, board: board)
+    Consolidate.create!(player: player, board: board)
+    Consolidate.create!(player: player, board: board)
+    Defend.create!(player: player, board: board)
+    Defend.create!(player: player, board: board)
+    DefendP.create!(player: player, board: board)
+    Raid.create!(player: player, board: board)
+    Raid.create!(player: player, board: board)
+    RaidP.create!(player: player, board: board)
+    Support.create!(player: player, board: board)
+    Support.create!(player: player, board: board)
+    SupportP.create!(player: player, board: board)
+  end
+
+  def create_power_tokens(player)
+    5.times do
+      PowerToken.create!(player: player)
+    end
+  end
+
+  def create_baratheon_cards(player)
+  end
+
+  def create_lannister_cards(player)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_lannister_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l156)